src/Action/Smtp.hs

```diff
@@ -98,6 +104,9 @@ sendMailToAddressIO logger receiver msg = do
         Left (e :: IOException) ->
             throwSendMailError $ IOErrorRunningSendMail e
   where
+    subjectLabel = case msg ^. msgSubjectLabel of
+        IdeaSpaceSubject is -> cs $ showIdeaSpace is
+        UserLoginSubject ul -> ul ^. unUserLogin
```

Comment:
1. I agree. I will keep that in mind.
2. It is only used here and in emails, no UI element is rendered from it.